src/doc/bootstrap

```diff
@@ -100,7 +100,16 @@ Sage depends.  It installs them automatically if it does not find
 equivalent system packages.
 
 EOF
-for PKG_BASE in $(sage-package list --has-file SPKG.rst :standard: | sort); do
+for PKG_BASE in $(sage-package list --has-file SPKG.rst :standard: | grep -v '^sagemath_' | sort); do
+    echo "* :ref:\`spkg_$PKG_BASE\`"
+done >> "$OUTPUT_INDEX"
+cat >> "$OUTPUT_INDEX" <<EOF
+
+Packages of the Modularized Sage Library
+----------------------------------------
+
```

Comment:
How about "Distribution Packages of the Sage Library", now that "distribution package" is a standard term?

Reply:
Good idea, done